apps/system/style/statusbar/statusbar.css

```diff
@@ -606,3 +611,11 @@ html[dir="rtl"] #statusbar-call-forwardings > * { float: right; }
 .utility-tray .sb-hide-operator .statusbar > .sb-icon-operator {
   visibility: hidden;
 }
+
+/* RTL */
+
+html[dir="rtl"] .statusbar {
```

Comment:
The RTL CSS guidelines also specifies that this rule should be near its LTR counterpart. That eases reviews and allows them not to be forgotten. 

But anyway, here you can use `offset-inline-start` again in the original rule and get rid of this :-) Thanks!